app/models/organization.rb

```diff
@@ -48,15 +47,21 @@ def github_organization
     @github_organization ||= GitHubOrganization.new(github_client, github_id)
   end
 
-  def slugify
-    self.slug = "#{github_id} #{title}".parameterize
+  def name_for_slug
+    "#{github_id} #{title}"
   end
 
   def one_owner_remains?
     users.count == 1
   end
 
+  def last_classroom_on_org?
+    Organization.where(github_id: github_id).length <= 1
```

Comment:
I understand what this method does but it's not super apparent. Maybe could you add a comment above describing its behavior?

Reply:
Added a comment describing the behavior 👍